cordformation/src/main/kotlin/net/corda/plugins/Node.kt

```diff
@@ -292,6 +292,8 @@ open class Node @Inject constructor(private val project: Project) {
         config.getInt("sshd.port")
     }
 
+    var usesDefaultSSHPort: Boolean = false
+
```

Comment:
Alternatively, you can declare the `useDefaultSSHPort` property as `internal` and annotate it as `@Internal` just to make the point:
```kotlin
@get:Internal
internal var usesDefaultSSHPort: Boolean = false
```
But these kind of gymnastics are telling me that the `Dockerform` task is becoming too complicated because it's just not obvious what is happening or needs to be happening at each stage.

Reply:
tried to move the usesDeafultSSHPort into the Dockerform instead of Node class. However, one of the issues that I had was I needed to update the node.conf of each generated node.

Reply:
Fair enough - this _is_ supposed to be a small and quick PR :grinning:.